lsp-metals.el

```diff
@@ -231,6 +231,15 @@ change it again."
   (interactive)
   (lsp-send-execute-command "reset-choice" ()))
 
+(defun lsp-metals-copy-worksheet-output ()
+  "Copy worksheet with evaluated results as comments."
+  (interactive)
+  (let ((command-result (lsp-send-execute-command "metals.copy-worksheet-output" (lsp--buffer-uri))))
+    (if (ht-get command-result "value")
```

Comment:
we currently support running with both plists/hts as a json deserialization target. This means that you should either use lsp-get or use lsp-interface like this: https://github.com/emacs-lsp/lsp-metals/blob/3dd96c1a72012735ac2874b89461f7d65b58fa8a/lsp-metals-protocol.el

Reply:
fixed ;)